python/tvm/script/parser/tir/operation.py

```diff
@@ -46,12 +46,12 @@ def r(op: Type, i: int, m: OpMethod):  # pylint: disable=invalid-name
 
     for i in [0, 1]:
         # Case 1. binop
```

Comment:
i was thinking if we could remove all the binary operators here given our printer has taken good care of it

Reply:
Sure! All those overloaded op have been removed.